seoul_18_kimjeonguk/BOJ/SILVER III/11478 - 서로 다른 부분 문자열의 개수/Q11478.java

```diff
@@ -0,0 +1,21 @@
+import java.io.*;
+import java.util.HashSet;
+import java.util.Set;
+
+public class Q11478 {
+    private static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+
+    public static void main(String[] args) throws IOException {
+        String s = br.readLine();
+
+        Set<String> set = new HashSet<>();
+
+        for (int i = 0; i < s.length(); i++) {
+            for (int j = i + 1; j <= s.length(); j++) {
+                set.add(s.substring(i, j));
+            }
+        }
```

Comment:
후위연산자보다 전위연산자를 사용하는게 빠르고 안전하다고 들었는데 어떻게 생각하시나요?
```suggestion
        for (int i = 0; i < s.length(); ++i) {
            for (int j = i + 1; j <= s.length(); ++j) {
                set.add(s.substring(i, j));
            }
        }
```